### PR TITLE
Adopt dotnet/arcade

### PR DESCRIPTION
**Pull request by sharwell**

### CODE REVIEW

**Review by ryanbrandenburg — APPROVED** [review]
A couple of questions and nit-picks but nothing to block on. Looks great!

File: src/PowerTools/DbContextPackage.cs [hunk 1]
```diff
@@ -23,12 +25,12 @@ namespace Microsoft.DbContextPackage
...
 
-    [PackageRegistration(UseManagedResourcesOnly = true)]
+    [PackageRegistration(AllowsBackgroundLoading = true, UseManagedResourcesOnly = true)]
     [InstalledProductRegistration("#110", "#112", "0.9.2", IconResourceID = 400)]
     [ProvideMenuResource("Menus.ctmenu", 1)]
     [Guid(GuidList.guidDbContextPackagePkgString)]
-    [ProvideAutoLoad("{f1536ef8-92ec-443c-9ed7-fdadf150da82}")]
-    public sealed class DbContextPackage : Package
+    [ProvideAutoLoad("{f1536ef8-92ec-443c-9ed7-fdadf150da82}", PackageAutoLoadFlags.BackgroundLoad)]
+    public sealed class DbContextPackage : AsyncPackage
```
[the remark this answers is not in the document]
Comment (sharwell) [reply src/PowerTools/DbContextPackage.cs:33]: Yes, also there are build errors if we don't apply this change.

File: eng/Versions.props [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    <MicrosoftVSSDKBuildToolsVersion>17.0.5234</MicrosoftVSSDKBuildToolsVersion>
+    <MicrosoftXmlEditorPackageVersion>$(MicrosoftVisualStudioPackageCurrent)</MicrosoftXmlEditorPackageVersion>
+    <MoqPackageVersion>4.0.10827</MoqPackageVersion>
+    <MySqlDataEntityPackageVersion>6.7.2-beta-ef6</MySqlDataEntityPackageVersion>
+    <VsWebSiteInteropPackageVersion>$(MicrosoftVisualStudioPackageCurrent)</VsWebSiteInteropPackageVersion>
+    <wixVersion>3.11.2</wixVersion>
+    <XunitPackageVersion>1.9.2</XunitPackageVersion>
+
+    <!-- Libs -->
+    <!--<CommandLineParserVersion>2.2.1</CommandLineParserVersion>
```
[the remark this answers is not in the document]
Comment (sharwell) [reply eng/Versions.props]: Removed these and cleaned up the remaining ones.

File: setup/wix/EFToolsWillowMsi.wixproj [hunk 3]
```diff
@@ -18,8 +17,14 @@
     <MsiRuntimeInputsPackagesDir>$(OutDir)\MsiRuntimeInputs\packages</MsiRuntimeInputsPackagesDir>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(SingletonMsiName)' == '' ">
-    <SingletonMsiName>EF6Tools</SingletonMsiName>
+  <PropertyGroup>
+    <EmbedCab Condition="'$(EmbedCab)' == ''">no</EmbedCab>
+    <SingletonMsiName Condition="'$(SingletonMsiName)' == ''">EF6Tools</SingletonMsiName>
```
Comment (ryanbrandenburg) [inline setup/wix/EFToolsWillowMsi.wixproj:22]: Is there actually somewhere where we pass this in or is this just you trying to stay close to the previous behavior?

**Review by ryanbrandenburg — APPROVED** [review]

File: azure-pipelines.yml [hunk 4]
```diff
@@ -104,6 +104,7 @@ stages:
             -configuration $(_BuildConfig)
             -prepareMachine
             $(_InternalBuildArgs)
+            /m:1
```
[the remark this answers is not in the document]
Comment (sharwell) [reply azure-pipelines.yml:107]: Disables MSBuild parallelism.

File: setup/wix/EFToolsWillowMsi.wixproj (hunk 3 above)
Comment (sharwell) [reply setup/wix/EFToolsWillowMsi.wixproj:22]: Just keeping the original behavior

**Pull request closed without merging**